MainHospitalClass/Hospital.cs

```diff
@@ -2,15 +2,15 @@
 
 namespace MainHospitalClass
 {
-    public class Hospital
+    public class Hospital : Institution
     {
         private const int PerVisitPayment = 500;
         private string _name;
```

Comment:
You could create protected properties in Base class (for _budget as well) and use it here

Reply:
Good point 😅 Fixed.